src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -304,7 +304,7 @@ private MavenRepositoryImpl createRepository() throws Exception {
      * Walks over the plugins, record {@link #modules} and call {@link #discover(Module)}.
      */
     private void processPlugins(MavenRepository r) throws Exception {
-        ExecutorService svc = Executors.newFixedThreadPool(4);
+        ExecutorService svc = Executors.newFixedThreadPool(1);
```

Comment:
FWIW I can try reverting this one the PR build passes to see whether that can finish with 4 threads. If so, I won't care about local execution.